recipes/deepseek/DeepSeek-R1-Distill-Qwen-7B/grpo/config_base_math_smalllr.yaml

```diff
@@ -32,11 +32,12 @@ lr_scheduler_type: cosine
 max_prompt_length: 512
 max_completion_length: 1024
 max_steps: -1
+num_generations: 7
```

Comment:
implicitly implies that we use 7 training GPUs. Not ideal but any simple alternative?

Reply:
Seems ok given the improvements in memory usage.
FYI I ran their original config yesterday if you want to compare running the new recipe against it: 
https://wandb.ai/edbeeching/huggingface/runs/p3otw3m9?nw=nwuseredbeeching
(let me know if you cannot access the wandb report)

Reply:
cool, thanks!